src/ui/dialogs/settings.ts

```diff
@@ -79,7 +79,7 @@ class ClientSection extends Section {
 
 class AccountOverviewSection extends Section {
    constructor(navigation: Navigation) {
-      super(navigation, Translation.t('Accounts'));
+      super(navigation, false, Translation.t('Accounts'));
```

Comment:
Also adding a default parameter of `false` would better reflect the actual behaviour.

Reply:
done